code/game/objects/items/melee/misc.dm

```diff
@@ -131,6 +131,26 @@
 		user.death(FALSE)
 	REMOVE_TRAIT(src, TRAIT_NODROP, SABRE_SUICIDE_TRAIT)
 
+/obj/item/melee/sabre/bee
+	name = "the stinger"
+	desc = "Taken from a giant bee and forget one thousand time in pure honey. Its extremly light and sharp."
```

Comment:
```suggestion
	desc = "Taken from a giant bee and folded over one thousand time in pure honey. Can sting through anything."
```
Or you know, just `folded` instead of `forget`.

Reply:
"time" should also be changed to "times," just sayin

Reply:
good catch.